about.html

```diff
@@ -0,0 +1,109 @@
+                <li class="menu-item"><a href="#partner">Partners</a></li>
+                <li class="menu-item"><a href="#copy-rigths">Rights</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section class="intro" id="intro">
+            <img src="./images/mobile_hamburger.svg" alt="mobile_menu" class="mobile_menu">
+            <div class="container intro-main">
+                <h2 class="intro-title">Level Up at Node.js<br /> Code Camp 2023</h2>
+                <p class="intro-description intro-description-about"></p>
+                <div class="intro-contact">
+                    Please contact us per Email for nay further questions about Node.js Camp 2023!
+                    <a href="mailto:ahmedeid2684@gmail.com">ahmedeid2684@gmail.com</a>
+                </div>
+            </div>
+        </section>
+        <section class="logo" id="logo">
+            <div class="container logo-main">
+                <h3 class="logo-title">Creative Commons Global Summit 2015 Logo</h3>
+                <p class="lgoo-description">The logo of Node.js Camp 2023 was decided through the logo competition
+                    from
+                    8.June to 7.July.</p>
+                <img src="./images/logos/logo-no-background.png" alt="logo">
+            </div>
+        </section>
+        <section class="previous-camps" id="previous-camps">
+            <div class="container previous-camps-main">
+                <h3 class="camp-title">See the Previous Node.js Camps</h3>
+                <p>Take a look at the last two Local Previous Node.js Camp which took place in Egypt Cairo and in
+                    Ismailia</p>
+                <div class="camp-card">
+                    <figure>
+                        <img src="./images/prevCamp/camp1.jpg" alt="camp1">
+                        <figcaption><span>2022</span>Camp 1</figcaption>
+                    </figure>
+                    <figure>
+                        <img src="./images/prevCamp/camp2.jpg" alt="camp2">
+                        <figcaption><span>2023</span>Camp 2</figcaption>
+                    </figure>
+                </div>
+            </div>
+        </section>
+        <section class="partner" id="partner">
+            <div class="container">
+                <h3 class="partner-title">Partner</h3>
+                <ul class="partner-main">
+                    <li class="partner-item"><img src="./images/partners/Microverse.png" alt="Microverse logo"></li>
+                    <li class="partner-item"><img src="./images/partners/microsoft.png" alt="Microsoft logo"></li>
+                    <li class="partner-item"><img src="./images/partners/IBM.png" alt="IBM logo"></li>
+                    <li class="partner-item"><img src="./images/partners/Coursera.png" alt="Courser logo"></li>
+                    <li class="partner-item"><img src="./images/partners/codeCamp.png"></li>
```

Comment:
- [ ] Please 🙏🏽 use the `alt` attribute for the image for the best practices of HTML & CSS.